src/components/AppHero.vue

```diff
@@ -4,7 +4,7 @@
       <div class="hero__row">
         <div class="hero__column">
           <h1 class="hero__heading">
-            æternity {{ selectedNetwork }} at a glance
+            æternity at a glance
```

Comment:
Was it requested to change? If I remember correctly it was originally requested to be this way by Yani to give a user very visible information on which network is used.